frontend/express/app.js

```diff
@@ -654,11 +654,26 @@ app.use(function(err, req, res, next) { // eslint-disable-line no-unused-vars
 
 
 //prevent bruteforce attacks
-bruteforce.db = countlyDb;
-bruteforce.mail = countlyMail;
-bruteforce.paths.push(countlyConfig.path + "/login");
-bruteforce.paths.push(countlyConfig.path + "/mobile/login");
-app.use(bruteforce.defaultPrevent);
+preventBruteforce.db = countlyDb;
+preventBruteforce.mail = countlyMail;
+
+for (let path of ["/login", "/mobile/login"]) {
+    const absPath = countlyConfig.path + path;
+    preventBruteforce.pathIdentifiers[absPath] = "login";
+    preventBruteforce.userIdentifiers[absPath] = (req) => req.body.username;
+}
+
+preventBruteforce.blockHooks["login"] = function(uid, req, res) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['req' is defined but never used.](https://app.codacy.com/manual/ar2rsawseen/countly-server/pullRequest?prid=5182840)